src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -115,7 +115,7 @@ func bootstrap(fs afero.Fs) *cobra.Command {
 		configPath string
 	)
 	c := &cobra.Command{
-		Use:   "bootstrap [--self <ip>] [--ips <ip1,ip2,...>]",
+		Use:   "bootstrap [--self <ip>] [--ips <ip1,ip2,...>] [flags]",
```

Comment:
This is not needed, cobra will add the `[flags]` automatically :smile:

Reply:
Nice! done